src/transformStateWithClones.js

```diff
@@ -8,6 +8,32 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  let cloneState = Object.assign({}, state);
```

Comment:
You can change this with, but I think this code does exactly the same thing. Just noting for the sake of it.
```js
let cloneState = { ...state };
```

Reply:
And for the sake of it I will also do it.

Thank you Ummut!